Make the init script exit with status 2 when given an unknown command,
rather than with status 1 (#674534).  Exit code is already 2 upstream;
document LSB rationale.

diff --git a/scripts/oddjobd.in b/scripts/oddjobd.in
index eab4280..a314d0a 100644
--- a/scripts/oddjobd.in
+++ b/scripts/oddjobd.in
@@ -100,6 +100,7 @@ case "$1" in
 	;;
   *)
 	echo $"Usage: $0 {start|stop|restart|condrestart|try-restart|reload|force-reload|status}"
+	# LSB: exit 2 for invalid or excess arguments
 	exit 2
 esac
 
